Server/mods/deathmatch/logic/CPed.cpp

```diff
@@ -406,6 +406,9 @@ CVehicle* CPed::SetOccupiedVehicle(CVehicle* pVehicle, unsigned int uiSeat)
     static bool bAlreadyIn = false;
     if (!bAlreadyIn)
     {
+        if (uiSeat < MAX_VEHICLE_SEATS)
+            return m_pVehicle;
+
```

Comment:
Maybe drop that check altogether from there. Looks a bit weird to me.

Also, why that method even return `CVehicle`... It could return boolean. But probably refactoring it is a job for seperate PR.

Reply:
Checking from the vehicle's side only then?

Reply:
Yes.